client/van_serviceinterface_create.go

```diff
@@ -5,133 +5,22 @@ import (
 	"fmt"
 
 	"k8s.io/apimachinery/pkg/api/errors"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
 	"github.com/skupperproject/skupper/api/types"
-	"github.com/skupperproject/skupper/pkg/kube"
 )
 
-// TODO: Move to utils
-func stringifySelector(labels map[string]string) string {
-	result := ""
-	for k, v := range labels {
-		if result != "" {
-			result += ","
+func (cli *VanClient) VanServiceInterfaceCreate(ctx context.Context, service *types.ServiceInterface) error {
```

Comment:
An initial thought was to have types that are for api "interfaces" that may have a relation to but differ from internal types such that here would be the use of VanServiceInterfaceCreateOptions. The question is whether it is worth maintaining two separate types or consolidate, we can always come back and address later on.

Reply:
VanServiceInterfaceList already uses the ServiceInterface in its signature. I think using that same type as the return for VanServiceInterfaceInspect makes sense (but returning a single instance rather than a list of them). It then feels reasonably natural to use that as the input to VanServiceInterfaceCreate or VanServiceInterfaceUpdate.